content/en/index.md

```diff
@@ -7,163 +7,169 @@ url: /
 
-#### Zug 
-Es gibt diverse Zugverbindungen ~~von Kiel (ca. 1 Stunde) und~~ von Hamburg (ca. 2 Stunden) mit der Regionalbahn. Es gibt sogar häufig Mobiles Internet und Steckdosen in den Zügen. Gerade auf das Mobile Internet sollte man sich nicht verlassen und das Digital Detox genießen. Es fahren auch vereinzelt dänische IC's von Hamburg nach Flensburg. Es gibt eine tägliche EC Verbindung von Prag nach Flensburg. Für Gruppen, ohne 49€-Ticket, lohnt sich ab Hamburg das Schleswig-Holstein Ticket. Über Umwege könnt ihr euch Sparpreis Tickets kaufen, die sich aufgrund der hohen Regionalbahn-Kosten auch lohnen. Aus Dänemark könnt ihr mit der Dänischen Staatsbahn anreisen.
+#### Train
+There are various train connections ~~ from Kiel (approx. 1 hour) and ~~ from Hamburg (approx. 2 hours) by regional train. There is often even mobile internet and power sockets on the trains. You should not rely on mobile internet and enjoy the digital detox. Occasionally, Danish ICs also run from Hamburg to Flensburg. There is a daily EC connection from Prague to Flensburg. For groups without a €49 ticket, the Schleswig-Holstein ticket from Hamburg is worthwhile. You can buy Sparpreis tickets via detours, which are also worthwhile due to the high regional train prices. You can travel from Denmark with the Danish State Railway.
 
-#### Auto
-Das Hacks on the Beach könnt ihr einfach über die A7 und dann über die B200 erreichen. Ab Hamburg braucht ihr ungefähr 1 Stunde und 55 Minuten. Aus Sonderburg sind es ca. 40 Minuten und aus Kiel braucht ihr 1 Stunde.
-Es gibt viele Parkplätze in Laufweite (ca. 2 Minuten / 200 Meter), es gibt keine Parkplätze direkt an der Location. Das Abstellenden von Wohnmobilien ist dort in der Regel kein Problem. Es gibt dort aber keine Toiletten oder Duschen. Es gibt keine Dusche in der Veranstaltungslocation.
-Sidenote: Es gibt noch mehr Parkplätze, ob diese belegt oder Frei sind, kannst du in der App CityPilot sehen. 
+#### Car
 
-#### Sontiges
-In Flensburg gibt es die Möglichkeit, relativ günstig mit dem Taxi zu Reisen.
-Für Ambitionierte führt auch der Europäische Fernwanderweg 1 durch Flensburg.
+Hacks on the Beach is easy to reach via the A7 motorway and then the B200. From Hamburg you need about 1 hour and 55 minutes. From Sonderburg it is approx. 40 minutes and from Kiel approx. 1 hour.
+There are numerous parking spaces within walking distance (approx. 2 minutes / 200 metres), but no parking spaces directly at the venue. Caravan parking is generally not a problem. Toilets and showers are not available. There are no showers at the venue.
+Note: There are additional parking spaces, you can see whether they are occupied or available in the CityPilot app.
```

Comment:
Link zu CityPilot?

Reply:
Habe ich verlinkt in DE und EN sowie für Apple und Android